Makefile

```diff
@@ -17,7 +17,7 @@ linux_compile:
 .PHONY: compile
 compile:
 	mkdir -p ./bin
-	go build -o bin/flytepropeller ./cmd/controller/main.go
+	go build -o bin/flytepropeller ./cmd/controller/main.go && cp bin/flytepropeller ${GOPATH}/bin
```

Comment:
Let's revert this... if you want, you can add an `install` Makefile target... and if so it should do `go install`... let's not make assumptions about where the go bin will be...